### PR TITLE
Document retrieving and usage of QDR CA cert

### PR DESCRIPTION
**Pull request by paramite**
Adds documentation of retrieval peocess of CA certificate for QDR on server side\
and it's usage on client side.

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+[id="getting-ca-certificate-from-stf-for-overcloud-configuration_{context}"]
+= Getting CA certificate from {Project} for overcloud configuration
+
+[role="_abstract"]
+To successfuly connect your {OpenStack} overcloud to {Project} you will first need to get CA certificate of {MessageBus} running on {Project} side.
+
+[[retrieving-the-server-side-ca-certificate-for-qdr]]
+== Retrieving the server side CA certificate for {MessageBus}
+
+To get a list of available certificates on {Project} side run the following command:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc:32]:
```suggestion
To get a list of available certificates in {Project} run the following command:
```

File: doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+[source,bash,options="nowrap",subs="verbatim"]
+----
+$ oc get secrets
+----
+
+There should be an item named *default-interconnect-selfsigned*. To get it's content, use following command:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc:39]:
```suggestion
Retrieve and note the content of the `default-interconnect-selfsigned` Secret:
```

File: doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+[source,bash,options="nowrap",subs="verbatim"]
+----
+$ oc get secret/default-interconnect-selfsigned -o jsonpath='{.data.ca\.crt}' | base64 -d
+----
+
+
+[[using-ca-certificate-content-in-red-hat-openstack-platform-configuration]]
+== Using CA certificate content in {OpenStack} configuration
+
+The resulting value has to be used in {OpenStack} configuration the following way:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc:50]:
```suggestion
Add the content from the `default-interconnect-selfsigned` Secret to the {OpenStack} configuration in `stf-connectors.yaml` to the `caCertFileContent` parameter within `MetricsQdrSSLProfiles`:
```

**Review by leifmadsen — COMMENTED** [review]
Made some suggested changes. Needs final review and adjustment from @JoanneOFlynn2018 but the content looks good to me.

**Review by lnatapov — APPROVED** [review]
The procedure is correct.

File: doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="getting-ca-certificate-from-stf-for-overcloud-configuration_{context}"]
+= Getting CA certificate from {Project} for overcloud configuration
+
+[role="_abstract"]
+To connect your {OpenStack} overcloud to {Project} you need to obtain the CA certificate of {MessageBus} running within {Project}.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_getting-ca-certificate-from-stf-for-overcloud-configuration.adoc:27]:
```suggestion
To connect your {OpenStack} overcloud to {Project}, you need to obtain the CA certificate of {MessageBus} that runs within {Project}.
```

**Comment by paramite** [comment]
Seems like we can merge this now, correct?

**Comment by JoanneOFlynn2018** [comment]
> Seems like we can merge this now, correct?

Hi Martin, 
I have to put a v13 wrapper on this yet, will get to it in about 2 hours time!

**Comment by leifmadsen** [comment]
> > Seems like we can merge this now, correct?
> 
> Hi Martin,
> I have to put a v13 wrapper on this yet, will get to it in about 2 hours time!

@paramite 

Note that I mentioned to Joanne today that we need to wrap this configuration in our ifdef 13 stuff because these changes are not applicable for OSP16 (yet). We do need to make sure we expose this across both OSP13 and OSP16.x (and 17) once the changes are available everywhere, especially once the OSP16 work results in the same QDR version appearing there that caused the 13 regression.

**Pull request merged**